marketplace-client/public/main.js

```diff
@@ -3,10 +3,31 @@ $(document).ready(function(){
   $("#header").sticky({topSpacing:0, zIndex: '50'});
   //scroll to top on route
   window.scrollTo(0, 0);
+  clock();
 });
 
+function clock() {
+  var d        = new Date();
+  var date     = d.getDate();
+  var year     = d.getFullYear();
+  var month    = d.getMonth();
+  var monthArr = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November","December"];
+  month        = monthArr[month];
+  $("#currentDate").val(date + " " + month + ", " + year);
+}
+
 !(function($) {
   "use strict";
+
+
+  //signout on click clear localstorage, reload page
+  $(document).on('click', '#signout', function(e) {
+    if(localStorage.getItem('token') !== null){
+      localStorage.clear();
+      window.location.reload()
```

Comment:
Redirect to home page on sign out instead. It takes you back to the page you were accessing before sign out and it ends up being blank since you no longer have access to the page.